themes/micromdm/static/css/micromdm.css

```diff
@@ -43,11 +43,12 @@ p {
 }
 
 section {
-  min-width: 100%;
+  min-width: 1280px;
```

Comment:
what happens with screens that do not have this resolution? 
I'm confused about specific px width vs using `%`